### ISSUE TITLE
Comma Separated Properties

### ISSUE THREAD

**Issue by TheColorRed** [HvyIndustries/crane#152]
Comma separated properties are read as one property.

```php
<?php

class Test {

    private $one = '123', $two = '456';

}
```

The parser or server only knows that the property `$one` exists, but not property `$two`.

**Comment by nevadascout** [comment]
Want to have a go at fixing this @TheColorRed? Or shall I add it to my list?

**Comment by TheColorRed** [comment]
I can look into it.

**Comment by TheColorRed** [comment]
Okay I have made the changes, I will create a PR.

**Comment by nevadascout** [comment]
Fixed in PR #156
